README.md

```diff
@@ -21,53 +20,47 @@ Below are instructions to quickly install and configure nginMesh.  Currently, on
 ### Prerequisites
 Make sure you have a Kubernetes cluster with at least 1.9 or greater due to fact only automatic sidecar injection is supported and no alpha feature is enabled. Please see [Prerequisites](https://istio.io/docs/setup/kubernetes/quick-start.html) for setting up a kubernetes cluster.
 
-### Installing Istio and nginMesh
+### Install Istio and nginMesh
 nginMesh requires installation of Istio first.
 
 1. Download and install Istio 0.7.1:
 ```
```

Comment:
Please indent all paragraphs with commands. For example:

Before:
1. Download and install Istio 0.7.1:
 ```
curl -L https://git.io/getLatestIstio | ISTIO_VERSION=0.7.1 sh -
```
After:
1. Download and install Istio 0.7.1:
    ```
   curl -L https://git.io/getLatestIstio | ISTIO_VERSION=0.7.1 sh -
   ```